src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedConversionSymbol.cs

```diff
@@ -42,11 +42,12 @@ internal sealed class SourceUserDefinedConversionSymbol : SourceUserDefinedOpera
             BindingDiagnosticBag diagnostics) :
             base(
                 MethodKind.Conversion,
+                explicitInterfaceType: null,
```

Comment:
Is explicit implementation of `static abstract` conversion operators coming later?

Reply:
> Is explicit implementation of static abstract conversion operators coming later?

Yes, they aren't supported in interfaces at the moment, no point to test/implement their implementation yet.